recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java

```diff
@@ -29,4 +34,20 @@ public JavaGenerator(Cookbook cookbook) {
             new JavaGetterFilter()
         );
     }
+
+    @Override
+    protected String renderTemplate(String templatePath, Map<String, Object> data) {
+        String generatedJava = super.renderTemplate(templatePath, data);
+
+        Formatter formatter = new Formatter(JavaFormatterOptions.builder().style(JavaFormatterOptions.Style.AOSP).build());
```

Comment:
You prefer AOSP over the standard google one? idk the differene...

Reply:
I just saw that AOPS had 4 space indents while google had 2 space indents.